test/Test/Prolens/Property.hs

```diff
@@ -61,8 +61,27 @@ profunctorsSpec = describe "Profunctor" $ do
             eqFun
                 (dimap (ab . bc) (yz . xy) f)
                 (dimap bc yz $ dimap ab xy f)
+    describe "Forget" $ do
+        it "Identity: dimap id id ≡ id" $ hedgehog $ do
+            f <- forAllWith (const "f") genForget
+            eqForget (dimap id id f) f
+        it "Composition: dimap (ab . bc) (yz . xy) ≡ dimap bc yz . dimap ab xy" $ hedgehog $ do
+            f  <- forAllWith (const "f")  genForget
+            ab <- forAllWith (const "ab") genFunction
+            bc <- forAllWith (const "bc") genFunction
+            xy <- forAllWith (const "xy") genFunction
+            yz <- forAllWith (const "xy") genFunction
+
+            eqForget
+                (dimap (ab . bc) (yz . xy) f)
+                (((dimap bc yz) . (dimap ab xy)) f)
```

Comment:
You can reduce the amount of parenthesis to make it clearer 🙂 
```suggestion
                ((dimap bc yz . dimap ab xy) f)
```

Reply:
Sure thing @vrom911